### PR TITLE
issue 657

### PR DESCRIPTION
**Pull request by nmccready**
\#657 
attempt to fix template compile on label and popup/label w cluster

### CODE REVIEW

**Comment by nmccready** [comment]
@zacronos @tombatossals @jessertaylor

File: dist/angular-leaflet-directive.js [hunk 1]
```diff
@@ -1966,6 +1968,14 @@ angular.module("leaflet-directive")
                     });
                 }
             });
+            marker.on("add", function(/* event */) {
+                if (watching) {
+                    safeApply(leafletScope, function() {
```
Comment (nmccready) [inline dist/angular-leaflet-directive.js]: label._container at this point should be defined

**Comment by zacronos** [comment]
LGTM

**Comment by jessertaylor** [comment]
nice work

**Pull request merged**